core/templates/pages/learner-dashboard-page/learner-groups-tab.component.ts

```diff
@@ -64,6 +64,7 @@ export class LearnerGroupsTabComponent {
         (learnerDashboardLearnerGroups) => {
           this.learnerGroupsJoined = (
             learnerDashboardLearnerGroups.learnerGroupsJoined);
+          console.error(this.learnerGroupsJoined, 'group joined bhole');
           this.invitedToLearnerGroups = (
```

Comment:
Please evaluate your work before requesting a review. here and elsewhere!!.

Reply:
Removed
Thanks!